docs/discover/set-time-filter.asciidoc

```diff
@@ -16,7 +16,8 @@ To set a time filter with the Time Picker:
 image::images/time-filter.jpg[Time filter shortcuts]
 . To specify a time filter relative to the current time, click *Relative* and
 specify the start time as a number of seconds, minutes, hours, days,
-months, or years ago.
+months, or years ago (or from now). You can also specify the end time relative
```

Comment:
I think I'd explicitly say that relative times can be past or future. 

To specify a time filter relative to the current time, click *Relative* and specify the start time as a number of seconds, minutes, hours, days, months, or years. You can also specify the end time relative to the current time. Relative times can be in the past or future.